src/calculateRentalCost.js

```diff
@@ -29,6 +29,17 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const count = days * 40;
+
+  if (days >= 3 && days < 7) {
+    return count - 20;
```

Comment:
usually called `magic numbers` - 20 and 50 are kinda - what are those?
generally same with 40 
declare variables with explanatory names and use them in code so that code is a readme of its own